### PR TITLE
Test zbar to be able to decode a qr code

### PR DESCRIPTION
**Pull request by DimStar77**

### CODE REVIEW

**Comment by okurz** [comment]
Could not verify it works: http://lord.arch/tests/1733
I am working on it

**Comment by okurz** [comment]
please merge #1492 instead or patch with
```perl
diff --git a/tests/console/zbar.pm b/tests/console/zbar.pm
index f92ca6c..62fa52a 100644
--- a/tests/console/zbar.pm
+++ b/tests/console/zbar.pm
@@ -13,13 +13,10 @@ use testapi;
 use utils;
 
 sub run() {
-    my $self = shift;
-    select_console 'root-console';
-
-    assert_script_run("zypper -n in zbar");
     select_console 'user-console';
+    assert_script_sudo("zypper -n in zbar");
     assert_script_run("wget " . autoinst_url . "/data/qr.png -O /tmp/qr.png");
-    validate_script_output "zbarimg /tmp/qr.png", sub { m/OPENPGP4FPR:6F58C4635A519E8C4A6ACD6EE69F22089B497C99?mac=891507df5bf02c60452e0de63ec30c24/ };
+    validate_script_output "zbarimg /tmp/qr.png", sub { m/OPENPGP4FPR:6F58C4635A519E8C4A6ACD6EE69F22089B497C99\?mac=891507df5bf02c60452e0de63ec30c24/ };
 }
```

So, yes, the question mark needs to be escaped :-)

File: tests/console/zbar.pm [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
```
Comment (coolo) [inline tests/console/zbar.pm]: had it on your disk for a while?

Comment (okurz) [inline tests/console/zbar.pm]: done

**Comment by okurz** [comment]
merged locally as discussed with @DimStar77

**Pull request closed without merging**